hieradata/hosts/test4.yaml

```diff
@@ -0,0 +1,140 @@
+role::trafficserver::enable_xdebug: true
+role::trafficserver::systemd_hardening: false
+
+# Define hosts. If not defined everything automatically
+# goes to mediawiki backends.
+role::trafficserver::mapping_rules:
+    - type: map
+      target: https://matomo.miraheze.org
+      replacement: https://mon2.miraheze.org
+    - type: map
+      target: /
+      replacement: https://backends.miraheze.org
+      params:
+          - '@plugin=/usr/lib/trafficserver/modules/tslua.so'
+          - '@pparam=/etc/trafficserver/lua/normalize-path.lua'
+          # decode    :  /  @  !  $  (  )  *  ,  ;
+          - '@pparam="3A 2F 40 21 24 28 29 2A 2C 3B"'
+          # encode    [  ]  &  '  +  =
+          - '@pparam="5B 5D 26 27 2B 3D"'
+          - '@plugin=/usr/lib/trafficserver/modules/tslua.so'
+          - '@pparam=/etc/trafficserver/lua/rb-mw-mangling.lua'
+          - '@plugin=/usr/lib/trafficserver/modules/tslua.so'
```

Comment:
A lot of extra config. Is each item necessary?

Reply:
Removed, thanks, done.